lib/node_modules/@stdlib/math/base/special/factorial2/README.md

```diff
@@ -0,0 +1,142 @@
+# Double Factorial
+
+> [Double Factorial][double-factorial-function] function.
+
+
+<section class="intro">
+
+The [double factorial][double-factorial-function] function may be defined as the product
+
+<!-- <equation class="equation" label="eq:double_factorial_function" align="center" raw="n!! = \prod_{k=0}^m (n-2k) = n(n-2)(n-4)\ldots" alt="Double factorial function definition"> -->
+
+<div class="equation" align="center" data-raw-text="n!! = \prod_{k=0}^m (n-2k) = n(n-2)(n-4)\ldots" data-equation="eq:double_factorial_function">
+    <img src="" alt="Double factorial function definition">
+    <br>
+</div>
+
+<!-- </equation> -->
+
+or according to the recurrence relation
+
+<!-- <equation class="equation" label="eq:factorial_recurrence_relation" align="center" raw="n! = \begin{cases}1 &amp; \textrm{if } n = 0,\\(n-1)! \times n &amp; \textrm{if } n > 1\end{cases}" alt="Factorial function recurrence relation"> -->
+
+<div class="equation" align="center" data-raw-text="n! = \begin{cases}1 &amp; \textrm{if } n = 0,\\(n-2)! \times n &amp; \textrm{if } n > 1\end{cases}" data-equation="eq:factorial_recurrence_relation">
+    <img src="" alt="Double factorial function recurrence relation">
+    <br>
+</div>
+
+<!-- </equation> -->
+
+Following the convention for an [empty product][empty-product], in all definitions,
+
+<!-- <equation class="equation" label="eq:zero_double_factorial" align="center" raw="0!! = 1" alt="Zero double factorial"> -->
+
+<div class="equation" align="center" data-raw-text="0! = 1" data-equation="eq:zero_double_factorial">
+    <img src="" alt="Zero factorial">
+    <br>
+</div>
+
+<!-- </equation> -->
+
+Note that the double factorial should not be confused with the factorial function iterated twice, written as (n!)! and not n!!
+
+The [double factorial][double-factorial-function] is implemented for integer values in terms of the [Factorial][@stdlib/math/base/special/factorial] function using the relation:
+
+<!-- <equation class="equation" label="eq:double_factorial_function_and_factorial" align="center" raw="(2n)!! = 2^n * n!" alt="Factorial function extension via the Factorial function"> -->
+
+<div class="equation" align="center" data-raw-text="(2n)!! = 2^n * n!" data-equation="eq:double_factorial_function_and_factorial">
+    <img src="" alt="Double factorial function extension via the factorial function">
+    <br>
+</div>
+
+The [Gamma][@stdlib/math/base/special/gamma] function extends the [double factorial][double-factorial-function] function to non-integer values.
+
+<!-- <equation class="equation" label="eq:double_factorial_function_and_gamma" align="center" raw="(2n-1)!! = \Gamma \left( \frac{(2n+1)}{2} \right) * \frac{2^n}{\sqrt{\pi}}" alt="Factorial function extension via the Gamma function"> -->
```

Comment:
I do not believe you need the `*` (asterisk) in your equation. This applies here and immediately below.